neqo-transport/src/send_stream.rs

```diff
@@ -542,8 +542,8 @@ impl SendStream {
 
     pub fn mark_as_lost(&mut self, offset: u64, len: usize, fin: bool) {
         if let Some(buf) = self.state.tx_buf_mut() {
-            buf.mark_as_lost(offset, len)
-        };
+            buf.mark_as_lost(offset, len);
+        }
```

Comment:
this looks weird... clearly we don't need the ; after the closing brace, but do we really instead need it after the statement, or not?